google/cloud/spanner/integration_tests/data_types_integration_test.cc

```diff
@@ -417,8 +417,9 @@ TEST_F(PgDataTypeIntegrationTest, WriteReadArrayJson) {
   auto result = WriteReadData(*client_, data, "ArrayJsonValue");
   {
     // TODO(#10095): Remove this when JSONB[] is supported.
-    auto matcher = testing_util::StatusIs(
-        StatusCode::kNotFound, testing::HasSubstr("Column not found in table"));
+    auto matcher = StatusIs(StatusCode::kNotFound,
```

Comment:
Or maybe not match anything about the string?  Is the ErrorInfo any better?

Reply:
> Or maybe not match anything about the string?

Maybe, but I think we're probably best off being more specific, particularly in the case of a temporary workaround.

> Is the ErrorInfo any better?

I did teach the logger how to try to extract a `ResourceInfo` on a `kNotFound` error, but there was nothing in the log in this case, so its presence is at least something that we can't depend on.  [Aside: We may want to think about `testing_util/status_matchers.h` support for looking into `Status` payloads.]

So, for better or worse, I think I'll leave it as-is for now.